app/mailers/school_mailer.rb

```diff
@@ -429,4 +430,26 @@ def school_preterm_reminder(school:, season:)
       },
     ).tag(:school_preterm_reminder).associate_with(school)
   end
+
+  def fip_provider_has_withdrawn_a_participant(withdrawn_participant:, induction_coordinator:)
+    partnership = Partnership.find_by(school: withdrawn_participant.school, cohort: withdrawn_participant.cohort)
+
+    email = template_mail(
+      PARTICIPANT_WITHDRAWN_BY_PROVIDER,
+      to: induction_coordinator.user.email,
+      rails_mailer: mailer_name,
+      rails_mail_template: action_name,
+      personalisation: {
+        name: induction_coordinator.user.full_name,
+        withdrawn_participant_name: withdrawn_participant.user.full_name,
+        school: withdrawn_participant.school.name,
+        delivery_partner: partnership.delivery_partner.name,
+        lead_provider: partnership.lead_provider.name,
+      },
+    )
+    email
+      .tag(:sit_fip_provider_has_withdrawn_a_participant)
+      .associate_with(induction_coordinator, as: :induction_coordinator)
+      .associate_with(withdrawn_participant, as: :participant_profile)
+  end
```

Comment:
Note: we really need to stop putting all the emails on the school_mailer... But that a bit of a refactor so let's skip it for now though...